README.md

```diff
@@ -0,0 +1,134 @@
+and install it into your environment.
+
+Next, add the following to your list of installed apps in Django settings:
+```python
+    INSTALLED_APPS = (
+        ... # your other installed apps
+        drf_yasg,
+        edx_api_doc_tools,
+    )
+```
+
+Then, in ``urls.py``:
+```python
+...
+from edx_api_doc_tools import make_api_info, make_docs_urls
+...
+api_info = make_api_info(title="Registrar API", version="v2")
+urlpatterns += make_docs_urls(api_info)
+```
+
+Your should now be able to load the Swagger UI in a browser at
+https://${your_service}/api-docs.
+Note that by default,
+documentation is only generated for views under the root path ``/api``.
+Generation for other views is possible but requires some extra configuration.
+
+Finally, you can enrich the generated documentation:
+```python
+...
+from edx_api_doc_tools import parameter, schema
+...
+class MyView(APIView):
+
+    @schema(
+        parameters=[
+            parameter('min_date', str, 'Filter response by minimum date.'),
+        ],
+        responses={
+            403: 'User lacks required permission.'
+            404: 'Resource does not exist.'
+            # Note that 2xx response is automatically figured out by drf-yasg,
+            # with reponse schema coming from the serializer, if declared.
+        },
+    )
+    def get(self, request):
+        """
+        Get the resource.
+
+        This docstring will be used in the OpenAPI spec, and *supports Markdown!*
+        """
+        ...
+```
+
+# Documentation
+
+Comphrehensive documentation is coming soon.
+For now, check out the `example/` directory,
+which shows an fake API using these tools to generate documentation.
+
+# License
+
+The code in this repository is licensed under the Apache Software License 2.0 unless
+otherwise noted.
+
+Please see `LICENSE.txt` for details.
+
+# How To Contribute
+
+Contributions are very welcome.
+
+Please read [How To Contribute](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst>)
+for details.
+
+Even though they were written with `edx-platform` in mind, the guidelines
+should be followed for Open edX code in general.
+
+The pull request description template should be automatically applied if you are creating a pull request from GitHub. Otherwise you
+can find it at [PULL_REQUEST_TEMPLATE.md](https://github.com/edx/api-doc-tools/blob/master/.github/PULL_REQUEST_TEMPLATE.md).
+
+The issue report template should be automatically applied if you are creating an issue on GitHub as well. Otherwise you
+can find it at [ISSUE_TEMPLATE.md](https://github.com/edx/api-doc-tools/blob/master/.github/ISSUE_TEMPLATE.md>).
+
+Reporting Security Issues
+-------------------------
+
+Please do not report security issues in public. Please email security@edx.org.
+
+Getting Help
+------------
+
+Have a question about this repository, or about Open edX in general?  Please
+refer to this [list of resources](https://open.edx.org/getting-help)
+if you need any assistance.
+
+
```

Comment:
:clap:

Reading the installation instructions makes me wonder if it would be possible to install this app as an Open edX plugin, like [figures](github.com/appsembler/figures) for instance. (this is not something to resolve right away, of course)

Reply:
I'm not 100% clear on what an Open edX plugin is - is that related to edX Platform Django plugins, or is it something separate?

If you think it's something worth doing, feel free to open an issue against this repository and I can factor it into the list of priorities for these tools.